web/styles/inbox.css

```diff
@@ -18,6 +18,14 @@
 
         .unread_count {
             opacity: 1;
+            /* 15px at  16px/1em */
+            font-size: 0.9375em;
+            /* 0 4px 1px at 16px/1em */
+            padding: 0 0.25em 1px;
+            border-radius: 3px;
+            color: var(--color-inbox-row-unread-count);
+            font-variant: all-small-caps;
+            font-weight: 400;
```

Comment:
Something seems to have gone wrong with the sizing of unread count pills; they should be consistent in width for a given number of digits. Note how the `1`s are misaligned with the rest:

![image](https://github.com/user-attachments/assets/6e57c308-c64b-4a88-a473-998c3cf8d591)

Reply:
Fixed in this update.

<img width="134" height="494" alt="image" src="https://github.com/user-attachments/assets/043da4e5-9313-48af-92d2-c2b7c5978761" />